supabase/migrations/20260818064144_fix_channels_noupdate_apikey_bypass.sql

```
@@ -0,0 +1,62 @@
+-- GHSA-ph9c-vwjq-pqhj: API-key PostgREST traffic must not bypass channels.noupdate().
```

SQLFluff (4.2.2)

[error] Line is too long (86 > 80).

(LT05)